src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -112,7 +125,7 @@ private static bool ModifiersRequireMetadataVirtual(DeclarationModifiers modifie
                 int isMetadataVirtualInt = isMetadataVirtual ? IsMetadataVirtualBit : 0;
 
                 _flags = methodKindInt | declarationModifiersInt | isExtensionMethodInt | isMetadataVirtualIgnoringInterfaceImplementationChangesInt | isMetadataVirtualInt;
-                _returnsVoid = returnsVoid;
+                _flags2 = returnsVoid ? ReturnsVoidBit : 0;
```

Comment:
>ReturnsVoidBit [](start = 40, length = 14)

Not setting ```ReturnsVoidIsSetBit```?

Reply:
Good catch, thanks.

---
In reply to: [296895556](https://github.com/dotnet/roslyn/pull/36152#discussion_r296895556) [](ancestors = 296895556)